connmgr/connmanager.go

```diff
@@ -418,6 +408,15 @@ func (cm *ConnManager) Connect(c *ConnReq) {
 	if atomic.LoadInt32(&cm.stop) != 0 {
 		return
 	}
+
+	//During the time we wait for retry there is a chance that
+	//this connection was already cancelled
+	if c.State() == ConnCanceled {
+		log.Debugf("Ignoring retry for "+
```

Comment:
also line can be unwrapped after the move here.

Reply:
fixed.